app/code/Bulbulatory/Recomendations/Controller/Adminhtml/Recommendations/Delete.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Bulbulatory\Recomendations\Controller\Adminhtml\Recommendations;
+
+use \Magento\Framework\App\Action\Context;
+use Magento\Framework\Controller\ResultFactory;
+use Bulbulatory\Recomendations\Model\ResourceModel\Recommendation\CollectionFactory;
+
+class Delete extends \Magento\Backend\App\Action
+{
+    public function __construct(
+        Context $context,
+        collectionFactory $collectionFactory
+    ) {
+        $this->_collectionFactory = $collectionFactory;
+        parent::__construct($context);
+    }
+
+    public function execute() 
+    {
+        $model = $this->_objectManager->create(\Bulbulatory\Recomendations\Model\Recommendation::class);
+        try {
+            $id = $this->getRequest()->getParam('recommendation_id');
+            $model->load($id);
+            $model->delete();
```

Comment:
Raczej nie powinno się stosować `_objectManager`, jest to coś czego najlepiej unikać. Najlepiej, aby za operacje na obiektach odpowiedzialne było `repository` i za jego pomocą były te operacje przeprowadzane:
http://vinaikopp.com/2017/02/18/magento2_repositories_interfaces_and_webapi/

Ewentualnie można skorzystać z factory, które automatycznie dla modelu się generuje, ale wtedy wciąż kontroler będzie odpowiedzialny za komunikację z bazą, a tak być nie powinno.
https://www.mageplaza.com/magento-2-module-development/factory-object-magento-2.html